hw-model/c-binding/examples/api/caliptra_fuses.h

```diff
@@ -0,0 +1,19 @@
+// Licensed under the Apache-2.0 license
+#ifndef CALIPTRA_FUSES_H
+#define CALIPTRA_FUSES_H
+
+#include "caliptra_api.h"
+
```

Comment:
Can you add a big warning stating that these are for use in simulation only? "SoC firmware MUST NOT have read access to fuse values; a hardware state machine should be used to send fuse values to Caliptra over the APB bus."

Reply:
done